pycozmo/client.py

```diff
@@ -436,3 +439,7 @@ def get_anim_names(self) -> set:
     @property
     def anim_names(self) -> set:
         return self.get_anim_names()
+
+    def play_audio(self, audio):
+        self.audio.play(audio)
+        return self.audio
```

Comment:
Oh, sorry, I missed this one. I included this to call wait_until_finished in my tests, but it's not necessary.

But related to this, do you think it would be a good idea to have an 'action' class similar to the one in the cozmo SDK that allows you to do things like wait until is finished or check the state of the action? It might be useful (do you already have it?)

Reply:
Yes, I think it makes sense long term. The current solution is a good start.